packages/theme/src/theme/_navbar.search.scss

```diff
@@ -10,25 +10,20 @@
 				content: '';
 				flex: 1 0 auto;
 			}
-
-			@media (min-width: $grid-float-breakpoint) {
-				padding-top: $navbar-padding-vertical;
```

Comment:
`$navbar-padding-vertical` is not used anymore. Could you delete it from _variables.scss ?